package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aind-metadata-entry",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
This is strange. The version in main is already `0.1.1`. You should be bumping it to `0.1.2`.

Reply:
The actual package.json file from main branch shows 0.1.1. Maybe it's comparing my changes which was branched off before Louise merged her PR.